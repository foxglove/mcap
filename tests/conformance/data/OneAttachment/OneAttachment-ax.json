{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Attachment",
      "fields": [
        ["create_time", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"]
      ]
    },
    {"type": "DataEnd", "fields": [["data_section_crc", "3051995293"]]},
    {
      "type": "AttachmentIndex",
      "fields": [
        ["create_time", "1"],
        ["data_size", "3"],
        ["length", "78"],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"],
        ["offset", "25"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "1358402382"],
        ["summary_offset_start", "0"],
        ["summary_start", "116"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["ax"]}}
}
