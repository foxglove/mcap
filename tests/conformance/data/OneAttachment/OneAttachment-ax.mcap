version https://git-lfs.github.com/spec/v1
oid sha256:0cef5a1aaa0310244560c0bfdef0207932e0f6f0805014e2504b6eda252f77d2
size 240
