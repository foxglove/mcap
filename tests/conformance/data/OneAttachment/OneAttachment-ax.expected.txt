Header library="" profile=""
Attachment content_type=application/octet-stream created_at=1 data=<010203> log_time=2 name=myfile
AttachmentIndex content_type=application/octet-stream data_size=3 length=78 log_time=2 name=myfile offset=25
Footer summary_offset_start=0 summary_start=116
